runtime/containers.go

```diff
@@ -279,6 +279,17 @@ func (t *ContainerTarget) WriteFileContents(ctx context.Context, contents string
 	return nil
 }
 
+func (t *ContainerTarget) Init(ctx context.Context, workDir string) error {
```

Comment:
Will this method get called as part of an interface? I don't understand how this gets called.

Reply:
Man... :sweat_smile:

How is this working then? I'll double check. I thought that Init would be called on initialization